### PR TITLE
Add new Minimum Availability setting for Radarr

### PR DESCRIPTION
**Pull request by rigrassm**
Implements passing the Minimum Availability with each movie request sent to Radarr. The setting is set in the Radarr client configuration from a predefined lost of options in a drop-down menu.

Version bump to 1.20.6

### CODE REVIEW

File: lib/collections/collections.js [hunk 1]
```diff
@@ -425,7 +425,7 @@ var settings = new SimpleSchema ({
     radarrQUALITYPROFILEID: {
         type: Number,
         label: "Download quality profile for Radarr",
-        defaultValue: 1,
+        defaultValue: '1',
```
Comment (lokenx) [inline lib/collections/collections.js]: Reason for changing to a string versus just a number, as the type is defined as a number, the schema expects a number--could cause issues on new installations.

**Comment by rigrassm** [comment]
Crap, good catch. I was testing something with schemas and forgot to change it back to a number.

File: lib/collections/collections.js (hunk 1 above)
Comment (rigrassm) [inline lib/collections/collections.js]: Give me a bit to get home so i can change that and the corresponding check in radarrMethods.js

Comment (rigrassm) [inline lib/collections/collections.js]: Ok, the fix has been committed and pushed. https://github.com/lokenx/plexrequests-meteor/pull/480/commits/4f54ac3ce48d3cebb4e29e8ae2529e40e8d13987

**Pull request merged**